languages/julia/exercises/concept/elyses-enchantments/.docs/hints.md

```diff
@@ -0,0 +1,33 @@
+## 1. Retrieve a card from a stack
+
+- `Vector` indices start at `0`.
+
+## 2. Exchange a card in the stack
+
+- `Vector`s are mutable, you can change their contents at any time.
```

Comment:
Maybe it would be better to do this without mutation first?

Reply:
Do you mean we should have a copying method first?

I can't think of many things to do with arrays other than indexing (and maths) that don't require mutation.

Reply:
I mean we could show how to do all these things with copying (possibly even with tuples instead of arrays?).